src/components/forms/DataTypes.ts

```diff
@@ -0,0 +1,48 @@
+import {SelectOptionWithKeyNameAndAbbreviation} from './SelectOptionInterface';
+
+
+export class DataTypes {
+    static CITE: SelectOptionWithKeyNameAndAbbreviation = {
```

Comment:
Why are these fields defined in addition to having them in the array below? It seems like their definition is redundant with the `key` field; if you instead defined them in the array, you could have a static method which returns the element given a `key`.

Reply:
It may seem like it, but we have a case where the key we're matching includes a "-".  We can't include that in the variable name, hence why I explicitly added the key field.
See GSMINUS in SecondaryDataUseTerms.ts